drivers/scheduler/k8s/k8s.go

```diff
@@ -1794,7 +1795,12 @@ func (k *K8s) createStorageObject(spec interface{}, ns *corev1.Namespace, app *s
 				log.Infof("Setting SC %s volumebinding mode to immediate ", obj.Name)
 			}
 		}
-
+		// Change Context only in case of vCluster tests
+		if vcluster.ContextChange {
+			log.Infof("Changing context to %v ", vcluster.UpdatedClusterContext)
+			err := vcluster.SwitchKubeContext(vcluster.UpdatedClusterContext)
+			log.FailOnError(err, fmt.Sprintf("Could not change context to %v", vcluster.UpdatedClusterContext))
```

Comment:
can we just return the err similar to other conditions in the below lines. let the test handle the error